index.js

```diff
@@ -418,6 +419,16 @@ exports.connect = function(config, intern, callback) {
   type = internals.mod.type;
 
   internals.interfaces.SeederInterface._makeParamArgs = dummy;
+  if( config.ssl ){
+    var keys = Object.keys(config.ssl);
+    for(var i=0 ; i < keys.length ; i++ ){
+      var key = keys[i];
+      var value = config.ssl[key];
+      if( value instanceof Object && value.path){
+          config.ssl[key] = fs.readFileSync(value.path);
```

Comment:
Why would you want to set i.e. `config.ssl.imarandomkeyandiamrandom.path`? And don't just set `config.ssl.path`?